pkg/services/searchV2/testdata/allowed_actions_scope_uids.golden.jsonc

```diff
@@ -16,26 +16,26 @@
+//  | Name: kind     | Name: uid      | Name: name              | Name: panel_type | Name: url                            | Name: tags               | Name: ds_uid              | Name: location | Name: allowed_actions                                                                                                                                                                                                                     |
+//  | Labels:        | Labels:        | Labels:                 | Labels:          | Labels:                              | Labels:                  | Labels:                   | Labels:        | Labels:                                                                                                                                                                                                                                   |
+//  | Type: []string | Type: []string | Type: []string          | Type: []string   | Type: []string                       | Type: []*json.RawMessage | Type: []json.RawMessage   | Type: []string | Type: []json.RawMessage                                                                                                                                                                                                                   |
+//  +----------------+----------------+-------------------------+------------------+--------------------------------------+--------------------------+---------------------------+----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+//  | folder         | ujaM1h6nz      | abc2                    |                  | /dashboards/f/ujaM1h6nz/abc2         | null                     | []                        |                | [{"kind":"folder","uid":"ujaM1h6nz","actions":["folders:read"]}]                                                                                                                                                                          |
+//  | dashboard      | 7MeksYbmk      | Alerting with TestData  |                  | /d/7MeksYbmk/alerting-with-testdata  | [                        | [                         | yboVMzb7z      | [{"kind":"dashboard","uid":"7MeksYbmk","actions":["dashboards:write"]},{"kind":"ds","uid":"datasource-1","actions":[]}]                                                                                                                   |
```

Comment:
@ArturWierzbicki -- this is a result of using the constant.  I think the change is good, but it is different

Reply:
I see the only difference is the type name (`ds` instead of `datasource`), not sure if that was intentional or not